packages/appeals-service-api/src/db/schema.prisma

```diff
@@ -10,8 +7,120 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model User {
-  id    Int    @id @default(autoincrement())
-  email String @unique
-  name  String
+// NOTE
+//
+// we use GUIDs for IDs (see https://learn.microsoft.com/en-us/sql/t-sql/data-types/uniqueidentifier-transact-sql?view=sql-server-ver16)
+// this is because these IDs may be used in URLs and it makes them harder to guess
+// while we don't rely on that for security, it adds an extra layer 
+// not everything needs this, but easier to make them all consistent and the increase in size (vs int) is negligible
+
+// AppealUser represents a user of our service
+// this may be an appellant, agent, or LPA employee
+model AppealUser {
+  id         String  @id @default(dbgenerated("newid()")) @db.UniqueIdentifier
+  email      String  @unique
+  // role may only be needed for lpaUser, and may not be populated otherwise
+  role       String? // appellant | agent | lpaUser
```

Comment:
Thinking again I guess this is correct and we do want a role on both AppealUser and AppealToUser as lpa users are not linked to appeals directly

However I guess this will just be lpauser | null 
at least to start with as an appeal user could be linked as either an appellant or an agent at the same time

Reply:
Replaced in favour of `isLpaUser`